integration_tests/tests/sdb.rs

```diff
@@ -1,15 +1,51 @@
 #![cfg(feature = "sdb")]
 extern crate rusoto_core;
 extern crate rusoto_sdb;
+extern crate env_logger;
 
-use rusoto_sdb::{SimpleDb, SimpleDbClient, ListDomainsRequest};
+use rusoto_sdb::{SimpleDb, SimpleDbClient, ListDomainsRequest, CreateDomainRequest, DeleteDomainRequest};
 use rusoto_core::Region;
 
 #[test]
 fn should_list_domains() {
+    let _ = env_logger::try_init();
     let client = SimpleDbClient::simple(Region::UsEast1);
     let request = ListDomainsRequest::default();
 
     let result = client.list_domains(&request).sync();
     println!("{:#?}", result);
 }
+
+#[test]
+fn roundtrip_test() {
+    let _ = env_logger::try_init();
+    let client = SimpleDbClient::simple(Region::UsEast1);
+    let test_domain = "rusoto_domain".to_string();
+
+    // Create domain
+    let create_domain_req = CreateDomainRequest {
+        domain_name: test_domain.clone(),
+    };
+    let create_result = client.create_domain(&create_domain_req).sync();
+    println!("create domain result: {:#?}", create_result);
+    assert!(create_result.is_ok());
+
+    // see domain in list (may take up to 10 seconds)
+    let request = ListDomainsRequest::default();
+    let result = client.list_domains(&request).sync();
+    println!("list domains result: {:#?}", result);
```

Comment:
Be careful not to forget result.**unwrap()** or similar. Don't want to make another pull request that, simultaneously, fixes and breaks the tests ;).

Reply:
Great catch, thanks!